extensions/amp-bind/0.1/validator-amp-bind.protoascii

```diff
@@ -59,6 +59,8 @@ tags: {  # <amp-state> with json child
     name: "id"
     mandatory: true
   }
+  # <amp-bind>
```

Comment:
I think for initialization (at page load) it's good to have two flavors since they are distinct use cases -- using `src` is meant for pages served by cache that require fresh data from origin.